ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Byron.hs

```diff
@@ -516,6 +518,9 @@ encodeByronBlock epochSlots =
 encodeByronHeaderHash :: HeaderHash (ByronBlock cfg) -> Encoding
 encodeByronHeaderHash = toCBOR
 
+encodeByronGenTx :: GenTx (ByronBlock cfg) -> Encoding
+encodeByronGenTx (ByronTx tx) = toCBOR (fmap (const ()) tx)
```

Comment:
I like to use `void` instead of `fmap (const ())`.

Reply:
done